### PR TITLE
[#IOPAE-376] API Regenerate service keys

### PR DESCRIPTION
**Pull request by giuseppedipinto**
Add new API for regenerate service keys.

#### List of Changes
- Added new _**regenerateSubscriptionKey**_ functionality on _apim-client_
- Added new _**RegenerateServiceKeys**_ AZ Function and handler
- Added related _unit tests_

#### Motivation and Context
We need to regenerate security keys for a given service.

#### How Has This Been Tested?
Unit tests

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/io-services-cms-webapp/RegenerateServiceKeys/function.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "bindings": [
+        {
+            "authLevel": "anonymous",
```
Comment (gunzip) [inline apps/io-services-cms-webapp/RegenerateServiceKeys/function.json:4]: are these function being called passing through the API gateway? I'm wondering if is there any reason on keeping them accessible without an host key (anonymous access)

Comment (giuseppedipinto) [reply apps/io-services-cms-webapp/RegenerateServiceKeys/function.json:4]: The reason is that we have not addressed this issue yet _(we will address it in the next few days)_ and we will do it for all functions.

File: apps/io-services-cms-webapp/src/webservice/controllers/__test__/regenerate-service-keys.test.ts [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+const aManageSubscriptionId = "MANAGE-123";
+const ownerId = "123";
+const primaryKey = "any-primary-key-value";
+const secondaryKey = "any-secondary-key-value";
+
+const anApimResource = {
+  id: "any-id",
+  name: "any-name",
+  _etag: "_etag",
+  ownerId,
```
Comment (giamma1295) [inline apps/io-services-cms-webapp/src/webservice/controllers/__test__/regenerate-service-keys.test.ts:52]: When apim is called the returned object as the property `ownerId` valued in this format "something/.../userId", is this handled correctly?

Comment (giamma1295) [inline apps/io-services-cms-webapp/src/webservice/controllers/__test__/regenerate-service-keys.test.ts:52]: When apim is called the returned object as the property ownerId valued in this format "something/.../userId", is this handled correctly?

**Review by giamma1295 — CHANGES_REQUESTED** [review]
Aside from the possible ownerId problem, this pr LGTM, so just check if the ownerId is handled correctly.

File: apps/io-services-cms-webapp/src/webservice/controllers/__test__/regenerate-service-keys.test.ts (hunk 2 above)
Comment (giuseppedipinto) [reply apps/io-services-cms-webapp/src/webservice/controllers/__test__/regenerate-service-keys.test.ts:52]: thanks for reporting, I checked it and the format is not relevant for the purpose of these unit tests ;-)

**Review by giamma1295 — APPROVED** [review]

**Pull request merged**